R/data.R

```diff
@@ -0,0 +1,78 @@
+#' Synthetic dataset of stochastic SIR system with known Rt
+#'
+#' A dataset produced by Katelyn Gostic as part of a published manuscript:
+#' Gostic, Katelyn M., et al. "Practical considerations for measuring the
+#' effective reproductive number, R t." PLoS computational biology 16.12 (2020):
+#' e1008409. The data are simulated from a stochastic SEIR compartment model.
+#'
+#' This synthetic dataset has a number of desirable properties: 1. The force of
+#' infection is manipulated by changing the Rt, allowing for sudden changes in
+#' infection dynamics. This behavior is a strong test of the flexibility of an
+#' Rt estimation framework 2. The realized Rt is known at each timepoint 3. The
+#' dataset incorporates a simple generation interval and a reporting delay
+#'
+#' In their manuscript, Gostic et al. benchmark the performance of a number of
+#' Rt estimation frameworks, providing practical guidance on how to use this
+#' dataset to evaluate Rt estimates.
+#'
+#' In practice, we've found that the amount of observation noise in the
+#' incidence and/or observed cases is often undesirably low for testing. Many
+#' empirical datasets are much noisier and models that are built with these
+#' settings in mind can perform poorly on this dataset or fail to converge. We
```

Comment:
Is this "as a result... (we manually add noise)". As in this data is different to the Gostic et al. one?

Reply:
Does this clear it up?

```suggestion
#' settings in mind can perform poorly on this dataset or fail to converge. To the original dataset, we add a new column to the Gostic, 2020 dataset with additional observation noise: `obs_incidence`. We
```

Reply:
You add a new column which the original column plus some noise? If so what about saying "plus" rather than "with the original incidence counts and additional observation noise". Or "with" if it's not plus

Reply:
Ok going with "with additional observation noise". It's not "plus" because it's a draw from a negative binomial so the error is multiplicative.